aws/index.ts

```diff
@@ -2,8 +2,10 @@ import * as pulumi from "@pulumi/pulumi";
 import * as k8s from "@pulumi/kubernetes";
 
 import * as config from "./config";
+import * as fs from "fs";
 import * as eks from "./eks";
 import * as util from "./util";
+import { ServiceSpecType } from "@pulumi/kubernetes/core/v1";
```

Comment:
You can remove this import and use `k8s.core.v1.ServiceSpecType` where necessary (see import on line `2`). This is what we are doing in most of the places around this file.

Reply:
> Instead, why not use this file as the source of truth, and read/parse it in index.ts? It is just regular ts code, and we can add a YAML parser as a dependency.

That's a good idea. I'll take a look at how to do it.

Reply:
Fixed. Now using a fully qualified name instead to be aligned with the current approach.